mappings/none/BlockInWorld.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS none/asp none/BlockInWorld
```

Comment:
There is probably a better name, but from what I can tell, BlockInWorld can represent either a cached block or a snapshot depending on the boolean liveState that is passed in.